src/js/components/molecules/card/card.ts

```diff
@@ -0,0 +1,25 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+import { ICardProps, ICard } from './card.models';
+
+@Component({
+    name: 'card',
+    template: require('./card.html'),
+})
+
+export default class Card extends Vue implements ICardProps{
+    @Prop({ type: String, default: '' })
+        block!: string;
+    @Prop({ type: Object, required: true, default: {} })
```

Comment:
If the prop is required, we don't need to use default value.
Can you please remove that?!